doc/Installation/Install-LibreNMS.md

```diff
@@ -39,15 +39,30 @@ Connect to the server command line and follow the instructions below.
     === "NGINX"
         ```
         dnf -y install epel-release
-        dnf install bash-completion composer cronie fping git ImageMagick mariadb-server mtr net-snmp net-snmp-utils nginx nmap php-fpm php-cli php-common php-curl php-gd php-json php-mbstring php-process php-snmp php-xml php-zip php-mysqlnd python3 python3-PyMySQL python3-redis python3-memcached python3-pip rrdtool unzip
+        dnf module reset php
+        dnf module enable php:7.3
+        dnf install bash-completion cronie fping git ImageMagick mariadb-server mtr net-snmp net-snmp-utils nginx nmap php-fpm php-cli php-common php-curl php-gd php-json php-mbstring php-process php-snmp php-xml php-zip php-mysqlnd python3 python3-PyMySQL python3-redis python3-memcached python3-pip rrdtool unzip
+        ```
+    Composer does not ship via the `dnf` package manager anymore, so you will have to install it "manually". To install it globally:
```

Comment:
We can just use composer_wrapper.php, it will fetch composer.phar from the web.  The distro packaged version is more of a fallback when fetch fails, usually a proxy situation.

Reply:
Updated docs to use manual (unpreferred) method as the fallback.